canvas/dom.js

```diff
@@ -0,0 +1,62 @@
+const canvas = document.getElementById('canvas');
+const context = canvas.getContext('2d');
+const bgColor = document.querySelector('.bg-color');
+const brushColor = document.querySelector('.color');
+const brushSize = document.querySelector('.size');
+
+let lastX = 0;
+let lastY = 0;
+let drawingFlag = false;
+let stopDrawing = new Event('mouseup');
+
+canvas.width = 700;
+canvas.height = 550;
+canvas.style.backgroundColor = bgColor.value;
+context.lineJoin = 'round';
+context.lineCap = 'round';
+context.lineWidth = brushSize.value;
+
+//Drawing function
+function draw(e) {
+  if (!drawingFlag) return;
```

Comment:
of course, it is ''вкусовщина", but please keep the curly blocks. in my opinion, it is a bad habit to have one-liner ifs. in case you need something to the if statement it is very easy to forget that there are no curly braces and make a nasty bug